d2/src/main/java/com/linkedin/d2/balancer/strategies/relative/StateUpdater.java

```diff
@@ -28,10 +28,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ConcurrentMap;
-import java.util.concurrent.ScheduledExecutorService;
-import java.util.concurrent.TimeUnit;
+import java.util.concurrent.*;
```

Comment:
this should expand each imports. (it's a code style setting in IntelliJ preferences you may need to fix)

Reply:
Oh. Got it. Updated. Thanks